### PR TITLE
Node 16 and NPM 8

### PR DESCRIPTION
**Pull request by zackkrida**
Previously, only production issues were keeping us on Node 14 and npm 6. This PR upgrades both and updates the local configuration for `volta` in the `engines` key of the package.json file.

This also builds a new `package-lock.json` with the updated deps.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -109,16 +109,15 @@
     "vue-jest": "^3.0.7"
   },
   "engines": {
-    "node": ">= 12.0.0",
-    "npm": ">= 6.0.0"
+    "node": "16.13.0",
+    "npm": "8.1.0"
```
Comment (dhruvkb) [inline package.json:113]: Should we be pinning `engines` this aggressively? I liked the old approach that only pinned the major version.

File: package.json [hunk 2]
```diff
@@ -109,16 +109,15 @@
...
   },
   "browserslist": [
     "> 1%",
     "last 2 versions",
     "not ie <= 8"
   ],
   "volta": {
-    "node": "14.17.5",
-    "npm": "7.21.0"
```
Comment (dhruvkb) [inline package.json:122]: Why aren't we pinning `npm`?

**Review by dhruvkb — COMMENTED** [review]
Nice to upgrade to the new LTS. I have a few non-blocking questions.

**Review by sarayourfriend — APPROVED** [review]
🚀

**Review by dhruvkb — APPROVED** [review]
LGTM!

**Pull request merged**